installr

```diff
@@ -15,7 +15,7 @@ make_description() {
     if echo $remote | grep -q -E '^[a-zA-Z0-9\.]+$'; then
         pkg=$remote
     else
-        pkg=`echo $remote | sed 's/^[^\/]*\///'`
+        pkg=`echo $remote | awk -F / '{print $(NF)}' | sed -e 's/[@].*//'
```

Comment:
Thanks! The closing back tick is missing here I believe. And also:

```
❯ remote=user/pkg/dir; echo $remote | awk -F / '{print $(NF)}' | sed -e 's/[@].*//'
dir
```

At least remotes assumes that the name of the package is the same as the name of the repo.

Reply:
Thank you for your feedback, you were right I forgot the back tick.

As for remotes assuming that the name of the package is the same as the name of the repo, I believe it was already the case using `repo/pkg`, no?
Do you think of a better implementation?

Reply:
Yes, what I meant is that the installr code assumes that the name of the package is the name of the repo. But your PR assumes that it is the name of the subdirectory, if a subdirectory is given. Which is not a reasonable assumption by itself, but it is not what the current R code uses.

Reply:
I agree with you. Maybe I misunderstand something but in your example

```bash
remote=user/pkg/dir; echo $remote | awk -F / '{print $(NF)}' | sed -e 's/[@].*//'
dir
```

Shouldn't `pkg` be inside a subdirectory, therefore

```bash
remote=user/subdir/pkg; echo $remote | awk -F / '{print $(NF)}' | sed -e 's/[@].*//'
pkg
```

works as expected? That's what h2o is doing (`h2o-3/h2o-r/h2o-package`) for instance.

Reply:
No, the R code assumes that the package name is the same as the repo name. I.e. `/user/pkg/subdir`.

Reply:
I see, unfortunately, I needed the other way around: in my work we don't have a github plan with many private repository, so all my packages are inside a single repo.
If you think of a better idea, I'd be glad to here it but you are right, my PR breaks the way installr is thought.

Reply:
That's fine. There is syntax to express that:

```
[<package>=][github::]<username>/<repository>[/<subdir>][<detail>]
```

so if the repository name is not the same as the package name, then you can specify the `<package>=` prefix, e.g. `h2oEnsemble=h2oai/h2o-3/h2o-r/ensemble/h2oEnsemble-package`. Btw. even for h2o, the subdirectory name is clearly not the same as the package name.

So maybe we want to support that syntax here?

Reply:
Oh that's great, didn't know this syntax! Will try it right away :)